tests/deployments/HorizonPhaseOneUpdate.t.sol

```diff
@@ -0,0 +1,269 @@
+        variableRateSlope1: 0,
+        variableRateSlope2: 0
+      }),
+      initialDeposit: 0
+    });
+
+    JAAA_TOKEN_LISTING_PARAMS = TokenListingParams({
+      aTokenName: 'Aave Horizon RWA JAAA',
+      aTokenSymbol: 'aHorRwaJAAA',
+      variableDebtTokenName: 'Aave Horizon RWA Variable Debt JAAA',
+      variableDebtTokenSymbol: 'variableDebtHorRwaJAAA',
+      isRwa: true,
+      hasPriceAdapter: true,
+      oracle: JAAA_PRICE_FEED_ADAPTER,
+      underlyingPriceFeed: JAAA_PRICE_FEED,
+      supplyCap: 24_760_000,
+      borrowCap: 0,
+      reserveFactor: 0,
+      enabledToBorrow: false,
+      borrowableInIsolation: false,
+      withSiloedBorrowing: false,
+      flashloanable: false,
+      ltv: 71_00,
+      liquidationThreshold: 78_00,
+      liquidationBonus: 100_00 + 9_00,
+      debtCeiling: 0,
+      liqProtocolFee: 0,
+      interestRateData: IDefaultInterestRateStrategyV2.InterestRateDataRay({
+        optimalUsageRatio: 0.99e27,
+        baseVariableBorrowRate: 0,
+        variableRateSlope1: 0,
+        variableRateSlope2: 0
+      }),
+      initialDeposit: 0
+    });
+  }
+
+  function test_eMode_USTB_Stablecoins() public override {
+    test_eMode_disabled(1, USTB_STABLECOINS_EMODE_PARAMS);
+  }
+
+  function test_eMode_USCC_Stablecoins() public override {
+    test_eMode_disabled(3, USCC_STABLECOINS_EMODE_PARAMS);
+  }
+
+  function test_eMode_USYC_Stablecoins() public override {
+    test_eMode_disabled(5, USYC_STABLECOINS_EMODE_PARAMS);
+  }
+
+  function test_eMode_JTRSY_Stablecoins() public override {
+    test_eMode_disabled(7, JTRSY_STABLECOINS_EMODE_PARAMS);
+  }
+
+  function test_eMode_JAAA_Stablecoins() public override {
+    test_eMode_disabled(9, JAAA_STABLECOINS_EMODE_PARAMS);
+  }
+
+  function test_eMode_disabled(
+    uint8 eModeCategory,
+    EModeCategoryParams memory params
+  ) internal view {
+    assertEq(pool.getEModeCategoryCollateralConfig(eModeCategory).ltv, params.ltv, 'emode.ltv');
+    assertEq(
+      pool.getEModeCategoryCollateralConfig(eModeCategory).liquidationThreshold,
+      params.liquidationThreshold,
+      'emode.liquidationThreshold'
+    );
+    assertEq(
+      pool.getEModeCategoryCollateralConfig(eModeCategory).liquidationBonus,
+      params.liquidationBonus,
+      'emode.liquidationBonus'
+    );
+    assertEq(pool.getEModeCategoryLabel(eModeCategory), params.label, 'emode.label');
+
+    uint128 collateralBitmap = pool.getEModeCategoryCollateralBitmap(eModeCategory);
+    uint128 recoveredCollateralBitmap = 0;
+    for (uint256 i = 0; i < params.collateralAssets.length; i++) {
+      uint256 reserveId = pool.getReserveData(params.collateralAssets[i]).id;
+      assertEq(
+        collateralBitmap.isReserveEnabledOnBitmap(reserveId),
+        false,
+        string.concat('emode.collateralAsset ', vm.toString(params.collateralAssets[i]))
+      );
+      recoveredCollateralBitmap = recoveredCollateralBitmap.setReserveBitmapBit(reserveId, false);
```

Comment:
this will not modify the bitmap at all right? I don't think this checks make sense. We can just assert `collateralBitmap=0` and `borrowableBitmap=0`, wdyt?